tools/src/bin/benchmark-clvm-cost.rs

```diff
@@ -273,6 +335,8 @@ const NESTING_BASE_COST: u32 = 4;
 const EXPONENTIAL_COST: u32 = 8;
 const LARGE_BUFFERS: u32 = 16;
 const ALLOW_FAILURE: u32 = 32;
+const LIST_LENGTH_COST: u32 = 64;
+const ALT_PER_BYTE_COST: u32 = 128;
```

Comment:
I think it would be good to add a comment here to explain what `ALT_` means. (and maybe even the descriptions I posted on keybase for the other flags)

Reply:
Done